cmd/mayactl/app/command/commands.go

```diff
@@ -38,6 +53,9 @@ func NewMayaCommand() *cobra.Command {
 	// add the glog flags
 	cmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
 
+	// add the api addr flag
+	cmd.PersistentFlags().StringVarP(&mapiserver.MAPIAddr, "mapiserver", "m", "", "Address of Maya-Apiserver")
```

Comment:
Address of Maya-Apiserver -> "Maya API Service IP address.  You can obtain the IP address using `kubectl get svc -n < namespace where openebs is installed >`

Reply:
Also include a flag for port. With default value specified as 5656.

Reply:
ok